packages/shorebird_cli/lib/src/archive_analysis/linux_bundle_differ.dart

```diff
@@ -8,8 +8,13 @@ class LinuxBundleDiffer extends ArchiveDiffer {
   /// {@macro linux_bundle_differ}
   const LinuxBundleDiffer();
 
+  bool _isDirectoryPath(String path) {
```

Comment:
package:path probably gives you some of this.  Also just using Directory(path).existsSync() should tell you if you're dealing with a path on disk (I think that checks the type? but if not I'm sure there is a way to check the type)

Reply:
Maybe? We're getting paths as strings from zip files, so I'm not sure `io` would help us much here.